kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala

```diff
@@ -39,18 +41,18 @@ private[v1] class OperationsResource extends ApiRequestContext {
     content = Array(new Content(
       mediaType = MediaType.APPLICATION_JSON)),
     description =
-      "Get an operation detail with a given session identifier and operation identifier")
+      "Get a statement event")
   @GET
-  @Path("{operationHandle}")
-  def getOperationDetail(
-      @PathParam("operationHandle") operationHandleStr: String): OperationDetail = {
+  @Path("{operationHandle}/statementEvent")
+  def getStatementEvent(
+      @PathParam("operationHandle") operationHandleStr: String): KyuubiStatementEvent = {
     try {
       val operation = backendService.sessionManager.operationManager
         .getOperation(parseOperationHandle(operationHandleStr))
-      OperationDetail(operation.shouldRunAsync, operation.isTimedOut, operation.getStatus)
+      KyuubiStatementEvent(operation.asInstanceOf[ExecuteStatement])
```

Comment:
This means we reduce the scope of the operation? If yes, I suggest keeping the old one and adding the new one. Otherwise, we should find a way to support other operations.

Reply:
What about add `KyuubiOperationEvent` to support other operations？ WDYT @yaooqinn @yanghua

Reply:
Maybe it's a solution.

Reply:
https://github.com/apache/incubator-kyuubi/pull/1540/files/a3981776cd4e769ec8cc4c69094b662105fa381d..03937c2ceedded105768dc89e495816cb029a23d#r766348428 as I suggested before, we can use KyuubiStatementEvent to support all operations

Reply:
> https://github.com/apache/incubator-kyuubi/pull/1540/files/a3981776cd4e769ec8cc4c69094b662105fa381d..03937c2ceedded105768dc89e495816cb029a23d#r766348428 as I suggested before, we can use KyuubiStatementEvent to support all operations

Got it , the name of `KyuubiStatementEvent` may cause confusion,  retain it or rename to `KyuubiOperationEvent`?

Reply:
I'm OK with renaming